src/GNDStk/Child/src/operator.hpp

```diff
@@ -284,3 +284,53 @@ inline auto operator+(
       kwd.top()
    );
 }
+
+
+
+// -----------------------------------------------------------------------------
+// Child || Child
+// Create a Child that represents one or the other of two children
+// -----------------------------------------------------------------------------
+
+// <void> || <void>
+// Return type is Child<void,...>
+template<
+   Allow ALLOW, class CONVERTER, class FILTER
```

Comment:
I really like how this now allows for a better syntax when specifying a variant child.

I've never liked the following syntax:
```
++Child<>("axis grid")
```
since it does not really convey the meaning of "look for many 'axis' or 'grid' nodes". Using a single string to store all variant type names is also not the best idea (what if I mistype and forget the space).

The new syntax like this:
```
++(Child<>("axis") || Child<>("grid"))
```
does a much better job at conveying this meaning properly. I like it.

Reply:
It's worth noting that you can still use the "name1 name2 ..." construct, in a single child, to get this effect, if you really want to do that. The `||` operator just does that for you. I *could* make `||` create some sort of special object, like perhaps a `tuple` that contains two or more `Child` objects. When I originally implemented all this (it was for handling `variant`), I chose to use `Child` as-is, with the spaces between names, because the alternative of using a special object, like a `tuple`, would require going through the entire code base and putting in additional overloads of all sort of functions that current accept only a `Child`. So, using `Child` under-to-hood serves as a minimally-disruptive implementation, and then the `||` operator is a nicety for users.